assemblies/assembly-audit-log.adoc

```diff
@@ -0,0 +1,32 @@
+[id="assembly-audit_log"]
+= Audit logging in {product}
+:context: assembly-audit-log
```

Comment:
Agree on the first two points and will apply that feedback. 

On the third point, I think including the product in the title does help orient the user in cases where they search for "audit log" in engines like Google because, if they get results for OCP docs, RHDH docs, and whatever else, it would be nice to choose _which_ doc set they want to dive into from there. But I agree that if you're already in RHDH and you know that's where you want to be, that including the product name can look a little superfluous. Not sure if there are currently any guidelines on this, but I'd like to know what is truly considered to be the best approach.